File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/schs/SchsSCH0201U99LayoutTimeListInfo.java
package nta.med.data.model.ihis.schs;

import java.util.Date;

public class SchsSCH0201U99LayoutTimeListInfo {

	private Date reserDate;
	private String fromTime;
	private String bunho;
	private String suname;
	private String hangmogCode;
	private String hangmogName;
	private String gwa;
	private String doctorName;
	private String actYn;

	public SchsSCH0201U99LayoutTimeListInfo(Date reserDate, String fromTime,
			String bunho, String suname, String hangmogCode,
			String hangmogName, String gwa, String doctorName, String actYn) {
		super();
		this.reserDate = reserDate;
		this.fromTime = fromTime;
		this.bunho = bunho;
		this.suname = suname;
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
		this.gwa = gwa;
		this.doctorName = doctorName;
		this.actYn = actYn;
	}

	public Date getReserDate() {
		return reserDate;
	}

	public void setReserDate(Date reserDate) {
		this.reserDate = reserDate;
	}

	public String getFromTime() {
		return fromTime;
	}

	public void setFromTime(String fromTime) {
		this.fromTime = fromTime;
	}

	public String getBunho() {
		return bunho;
	}

	public void setBunho(String bunho) {
		this.bunho = bunho;
	}

	public String getSuname() {
		return suname;
	}

	public void setSuname(String suname) {
		this.suname = suname;
	}

	public String getHangmogCode() {
		return hangmogCode;
	}

	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}

	public String getHangmogName() {
		return hangmogName;
	}

	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}

	public String getGwa() {
		return gwa;
	}

	public void setGwa(String gwa) {
		this.gwa = gwa;
	}

	public String getDoctorName() {
		return doctorName;
	}

	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}

	public String getActYn() {
		return actYn;
	}

	public void setActYn(String actYn) {
		this.actYn = actYn;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Schs/SchsSCH0201U99LayoutTimeListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Schs
{
    [Serializable]
	public class SchsSCH0201U99LayoutTimeListInfo
	{
		private String _reserDate;
		private String _fromTime;
		private String _bunho;
		private String _suname;
		private String _hangmogCode;
		private String _hangmogName;
		private String _gwa;
		private String _doctorName;
		private String _actYn;

		public String ReserDate
		{
			get { return this._reserDate; }
			set { this._reserDate = value; }
		}

		public String FromTime
		{
			get { return this._fromTime; }
			set { this._fromTime = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Suname
		{
			get { return this._suname; }
			set { this._suname = value; }
		}

		public String HangmogCode
		{
			get { return this._hangmogCode; }
			set { this._hangmogCode = value; }
		}

		public String HangmogName
		{
			get { return this._hangmogName; }
			set { this._hangmogName = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String ActYn
		{
			get { return this._actYn; }
			set { this._actYn = value; }
		}

		public SchsSCH0201U99LayoutTimeListInfo() { }

		public SchsSCH0201U99LayoutTimeListInfo(String reserDate, String fromTime, String bunho, String suname, String hangmogCode, String hangmogName, String gwa, String doctorName, String actYn)
		{
			this._reserDate = reserDate;
			this._fromTime = fromTime;
			this._bunho = bunho;
			this._suname = suname;
			this._hangmogCode = hangmogCode;
			this._hangmogName = hangmogName;
			this._gwa = gwa;
			this._doctorName = doctorName;
			this._actYn = actYn;
		}

	}
}